Taskfile.yml

```diff
@@ -58,7 +58,6 @@ tasks:
   solo:mirror-node:
     desc: solo mirror-node deploy with port forward on explorer
     cmds:
-      - npm run build
```

Comment:
We need to make a task that will run build and leave an artifact behind somewhere, perhaps in temp directory with the filename as a timestamp set at startup.  That way, we can run the build one time, but check before any call.  If you remove the npm run build, then we run the risk that they may make a call that runs on an older build of code and does not match the code they have checked out on their machine.

Reply:
reverted the change